packages/p2p-v2/src/components/Modals/PaymentMethods/styles.scss

```diff
@@ -0,0 +1,24 @@
+.p2p-v2-payment-methods-modal {
+    &__wrapper {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        gap: 2.4rem;
+        width: 44rem;
+    }
+    &__buttons {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        gap: 1.6rem;
+        width: 100%;
+        &--cancel {
+            background: transparent;
+            border: 2px solid #999999;
+
+            &:active:not([disabled]) {
+                border: 1px solid #999999;
+            }
```

Comment:
can't these be handled by `Button` component?

Reply:
The only `variant` that looks similar to the `p2p` cancel button is the `outlined` `variant`, but it's configurations are a bit different from the `p2p` cancel button. That's why I decided to override them here